src/lib/transaction/Signatures.svelte

```diff
@@ -1,18 +1,21 @@
 <script lang="ts">
     import { StrKey, xdr } from 'stellar-sdk';
-
+    import WalletLanguage from '../../helpers/WalletLanguage';
     export let signatures: xdr.DecoratedSignature[];
+
+    const language = new WalletLanguage();
```

Comment:
is there a way this can be made globally available? This is a transversal dependency -- absolutely every module will benefit from a language helper.